src/app/shared/components/navbar/navbar.component.html

```diff
@@ -51,6 +51,10 @@
       <mat-icon>launch</mat-icon>
       Geodatenshop
     </a>
+    <a class="navbar__item" href="https://gdi.zh.ch/portal/home/" mat-button target="_blank" rel="noopener noreferrer">
+      <mat-icon>launch</mat-icon>
+      KoPla
```

Comment:
Who or what is a _KoPla_? 😂
In all seriousness: This is not an issue with your PR - you've added this as planned. But maybe a tooltip or something should contain the full name.

Reply:
I added a tooltip, but it does feel a little weird to have it just for this one link. Especially since the link is now only visible in internal applications and the internal users apparently should know, who or what a kopla is

Reply:
I think we should have this either everywhere or nowhere. We can add this quite easily if it's necessary - for now I would remove the tooltip.

Reply:
is removed